scripts/pre_commit_linter.py

```diff
@@ -645,7 +645,8 @@ def _check_import_order(all_files):
     summary_messages = []
     all_files = [
```

Comment:
Can you rename this to `files_to_check` (see [here](https://github.com/apb7/oppia/blob/99e7ba833b37fbac94ec60b6dacfe6c43052a8f5/scripts/pre_commit_linter.py#L683))?

Reply:
Done. Thanks, @DubeySandeep!